generate_toc.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# adapted from https://github.com/npryce/adr-tools
+# a combination of a few scripts, _adr_title, _adr_generate_toc
+
+set -e
+
+args=$(getopt i:o:p: $*)
+set -- $args
+
+link_prefix=./decisions/
+
+eval "$(adr config)"
+
+cat <<EOF
+# Architecture Decision Records
+EOF
```

Comment:
I suggest simply making this
```suggestion
echo "# Architecture Decision Records"
```